app/models/audio_version.rb

```diff
@@ -72,5 +72,12 @@ def set_status
       self.status = INVALID
       self.status_message = errors
```

Comment:
You probably need a `return` here now, right?

Reply:
argh, yes!